src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -0,0 +1,164 @@
+package teammates.ui.controller;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import teammates.common.datatransfer.FeedbackSessionType;
+import teammates.common.datatransfer.attributes.CourseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.EmailType;
+import teammates.common.util.SanitizationHelper;
+import teammates.common.util.TimeHelper;
+
+public abstract class InstructorFeedbackAbstractAction extends Action {
+
+    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(
+            FeedbackSessionAttributes newSession, List<String> sendRemainderEmailsList) {
+        return this.extractFeedbackSessionDataHelper(newSession, sendRemainderEmailsList);
+    }
```

Comment:
Making it abstract isn't what you want then. What're you trying to do in those helper methods? You can move the common functionality up to the AbstractAction so that those child classes that need it can use it. Right now, that definition of the helper method in the AbstractAction isn't useful, or dangerous at worst.